### PR TITLE
Add support of symfony 3.4/4.0

### PR DESCRIPTION
**Pull request by gregurco**
Add symfony lts version and 3.4 to travis
Remove varci.yml

- [ ] Bug fix
- [x] New feature
- [ ] BC breaks
- [ ] Deprecations
- [x] Tests pass

Fixed tickets: [#150]
License: MIT

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,3 +33,4 @@ matrix:
     allow_failures:
         - env: SYMFONY_VERSION=dev-master
         - php: nightly
+        - php: 7.2
```
Comment (florianpreusner) [inline .travis.yml]: why allowing failures on 7.2?

Comment (gregurco) [reply .travis.yml]: because it's not released yet. It's RC6, probably stable enough but not released.

Comment (florianpreusner) [reply .travis.yml]: ok, so idea is to remove it once it is released?

Comment (florianpreusner) [reply .travis.yml]: Maybe we should add a comment to make it clear

Comment (gregurco) [reply .travis.yml]: yep :slightly_smiling_face: and some changes in this conf should be done when symfony 3.4/4.0 will be released

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
     - SYMFONY_VERSION=3.2.*
     - SYMFONY_VERSION=3.3.*
+    - DEPENDENCIES='dev' SYMFONY_VERSION='3.4.*@dev' # LTS
     - SYMFONY_VERSION=dev-master
 
-before_script:
+matrix:
+    exclude:
+        - php: 7.0
+        - env: SYMFONY_VERSION=dev-master # requires PHP ^7.1.3
```
Comment (florianpreusner) [inline .travis.yml]: 👍  nice!

File: .varci.yml [hunk 3]
```diff
@@ -1,28 +0,0 @@
-ruleset:
```
Comment (florianpreusner) [inline .varci.yml]: thanks for cleanup!

File: .travis.yml (hunk 2 above)
Comment (gregurco) [reply .travis.yml]: strange... it didn't exclude with `php: 7.0` but excluded with `php: 7` :confused: now it's ok... have a change to have 100% green builds :slightly_smiling_face:

File: src/DependencyInjection/EightPointsGuzzleExtension.php [hunk 4]
```diff
@@ -92,6 +92,7 @@ public function load(array $configs, ContainerBuilder $container)
 
             $client = new Definition($options['class']);
             $client->addArgument($argument);
+            $client->setPublic(true);
```
Comment (florianpreusner) [inline src/DependencyInjection/EightPointsGuzzleExtension.php]: 👍

**Comment by gregurco** [comment]
@florianpreusner I added symfony 3.4 but I forgot about 4.0... probably it be added to, yep?

**Comment by gregurco** [comment]
@florianpreusner I think it's ready for merging but something is wrong with SensioLabsInsight :slightly_frowning_face:

**Comment by florianpreusner** [comment]
SensioLabsInsight seems to work again.

**Pull request merged**